prusti-common/src/vir/ast/expr.rs

```diff
@@ -4,6 +4,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+use viper::AstFactory;
```

Comment:
Why did you add this?

Reply:
I think it was auto-completed when I tried things out. I removed it again.